src/Adapter/Category/CategoryDataProvider.php

```diff
@@ -81,7 +81,7 @@ public function getCategory($idCategory = null, $idLang = null, $idShop = null)
      *
      * @return array categories
      */
-    public function getNestedCategories($root_category = null, $id_lang = false, $active = true, $groups = null, $use_shop_restriction = true, $sql_filter = '', $sql_sort = '', $sql_limit = '')
+    public function getNestedCategories($root_category = null, $id_lang = false, $active = false, $groups = null, $use_shop_restriction = true, $sql_filter = '', $sql_sort = '', $sql_limit = '')
```

Comment:
If you change the default value, you change the comportement all time this function is called, I don't think is the good solution. Why not call the function with active = false ?